example/consul_client/consul_client_test.go

```diff
@@ -213,98 +213,98 @@ func TestDeregisterServiceInstance(t *testing.T) {
-		Host:  "127.0.0.1", //require user provide
-		Port:  8500,        //require user provide
-		Token: "",
-	}
-
-	sdkClient := shenyu_sdk_client.GetFactoryClient(constants.CONSUL_CLIENT)
-	client, createResult, err := sdkClient.NewClient(ccp)
-
-	assert.Nil(t, err)
-	assert.True(t, createResult)
-
-	scc := client.(*consul_client.ShenYuConsulClient)
-	//Create ShenYuConsulClient end
-	uuid1, _ := uuid.GenerateUUID()
-	uuid2, _ := uuid.GenerateUUID()
-	uuid3, _ := uuid.GenerateUUID()
-
-	//RegisterServiceInstance start
-	//init MetaDataRegister
-	metaData1 := &model.ConsulMetaDataRegister{
-		ServiceId: uuid1,
-		ShenYuMetaData: &model.MetaDataRegister{
-			AppName: "testMetaDataRegister1", //require user provide
-			Path:    "/your/path1",           //require user provide
-			Enabled: true,                    //require user provide
-			Host:    "127.0.0.1",             //require user provide
-			Port:    "8080",                  //require user provide
-			RPCType: "http",                  //require user provide
-		},
-	}
-
-	metaData2 := &model.ConsulMetaDataRegister{
-		ServiceId: uuid2,
-		ShenYuMetaData: &model.MetaDataRegister{
-			AppName: "testMetaDataRegister2", //require user provide
-			Path:    "/your/path2",           //require user provide
-			Enabled: true,                    //require user provide
-			Host:    "127.0.0.1",             //require user provide
-			Port:    "8181",                  //require user provide
-			RPCType: "http",                  //require user provide
-		},
-	}
-
-	metaData3 := &model.ConsulMetaDataRegister{
-		ServiceId: uuid3,
-		ShenYuMetaData: &model.MetaDataRegister{
-			AppName: "testMetaDataRegister3", //require user provide
-			Path:    "/your/path3",           //require user provide
-			Enabled: true,                    //require user provide
-			Host:    "127.0.0.1",             //require user provide
-			Port:    "8282",                  //require user provide
-			RPCType: "http",                  //require user provide
-		},
-	}
-
-	//register multiple metaData
-	registerResult1, err := scc.RegisterServiceInstance(metaData1)
-	assert.Nil(t, err)
-	assert.True(t, registerResult1)
-
-	registerResult2, err := scc.RegisterServiceInstance(metaData2)
-	assert.Nil(t, err)
-	assert.True(t, registerResult2)
-
-	registerResult3, err := scc.RegisterServiceInstance(metaData3)
-	assert.Nil(t, err)
-	assert.True(t, registerResult3)
-
-	time.Sleep(time.Second)
-
-	//get nodes
-	instanceDetail, err := scc.GetServiceInstanceInfo(metaData1)
-	nodes1, ok := instanceDetail.([]*model.ConsulMetaDataRegister)
-	assert.NotNil(t, nodes1)
-	assert.True(t, ok)
-	assert.Nil(t, err)
-
-	instanceDetail2, err := scc.GetServiceInstanceInfo(metaData2)
-	nodes2, ok := instanceDetail2.([]*model.ConsulMetaDataRegister)
-	assert.NotNil(t, nodes2)
-	assert.True(t, ok)
-	assert.Nil(t, err)
-
-	instanceDetail3, err := scc.GetServiceInstanceInfo(metaData3)
-	nodes3, ok := instanceDetail3.([]*model.ConsulMetaDataRegister)
```

Comment:
why remove this?

Reply:
The single test content is repeated with `TestEntireConsulFunction`. I am trying to pass ci.